src/main/resources/application.conf

```diff
@@ -33,6 +33,11 @@ docker {
   // dockerToken = ""
 }
 
+jobavoidance {
+  // Set to true in order to avoid running previously run jobs.
+  enabled = false
+}
+
```

Comment:
I'm a fan of defaulted properties. Instead of the program crashing if they don't specify this new property in application.conf, what about defaulting it to `false` over in package.scala.

```scala
import lenthall.config._
// ...
val avoidJobs = ConfigFactory.load.getConfig("jobavoidance").getBooleanOr("enabled")
```

I know @mcovarr previously mentioned this reads funny. At any (appropriate) time that you guys like, cleanup is appreciated.